changelog/_unreleased/2020-10-23-remove-endless-loop-in-promotion-code-generator.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Remove endless loop in promotion code generator
+author: Joshua Behrens
+author_email: code@joshua-behrens.de
+author_github: @JoshuaBehrens
+---
+# Administration
+* Fixed bug in promotion code generator to stop generating randomized codes when no further codes are possible by pattern definition
```

Comment:
We currently only use certain keywords for changelog entries, please have a look [here](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md#changelog-entries) and change the line accordingly.

Reply:
Could you add a test for these files so this has not to be checked by a human every time I fail to write an English sentence? :D

Reply:
Good point, there are some validations in the `ChangelogParser` already, but this indeed isn't checked at the moment. I've created a [ticket](https://issues.shopware.com/issues/NEXT-11801) to check this.

Reply:
@JoshuaBehrens Could you fix that please? 🙂

Reply:
Already 14 days ago? Thanks for the bump.

Reply:
I hope the message is now more useful :)